content/docs/latest/installing/vms/hyper-v.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Running Flatcar Container Linux on Hyper-V
+title: Running on Hyper-V
+weight: 30
+aliases:
+    - ../../os/booting-on-hyper-v
+    - ../../cloud-providers/booting-on-hyper-v
+---
+
+_While we always welcome community contributions and fixes, please note that Hyper-V is not an officially supported platform at this time. (See the [platform overview](/#installing-flatcar).)_
```

Comment:
```suggestion
_While we always welcome community contributions and fixes, please note that Hyper-V is not an officially supported platform at this time because the release tests don't run for it. (See the [platform overview](/#installing-flatcar).)_
```